packages/flutter/lib/src/rendering/sliver_persistent_header.dart

```diff
@@ -17,6 +17,33 @@ import 'sliver.dart';
 import 'viewport.dart';
 import 'viewport_offset.dart';
 
+/// A callback function that's invoked when the [SliverAppBar] is pulled into an
+/// overscroll and stretched to the offset specified by `stretchTriggerOffset`.
+/// Must return a [Future].
+typedef StretchCallback = Future<void> Function();
+
+/// Specifies how a stretched header is to trigger a [StretchCallback].
+///
+/// See also:
+///
+///  * [SliverAppBar], which creates a header that can stretched into an
+///  overscroll area and trigger a callback function.
+class OverScrollHeaderStretchConfiguration {
+  /// Creates an object that specifies how stretched header is to trigger a
```

Comment:
"how stretched header" => "how a stretched header"

I think... If that seems right to you.